.github/actions/install-rust-toolchain/action.yml

```diff
@@ -0,0 +1,35 @@
+## Inspired by: https://github.com/actions-rs/toolchain/issues/216#issuecomment-1291613319
+name: Install Rust toolchain
+description: "Install Rust toolchain, with caching"
+
+inputs:
+  toolchain:
+    required: false
+    description: "Rust toolchain, e.g. 'stable' or 'nightly'"
+    default: stable
+  cache-key:
+    required: false
+    description: "Extra key to resolve cache"
+    default: ""
+
+runs:
+  using: "composite"
+  steps:
+    - name: "Install Rust"
+      run: |
+        rustup toolchain install ${{ inputs.toolchain }} --profile minimal --no-self-update
+        rustup default ${{ inputs.toolchain }}
+      shell: bash
```

Comment:
In workflow files this can be set globally, is this also possible in composite flows?

Reply:
Yep, setting `shell` in `actions` is required:
<img width="729" alt="Screenshot 2022-10-27 at 13 29 17" src="https://user-images.githubusercontent.com/608862/198284401-219ba671-20a3-460d-ae4f-445db8a6afa7.png">